babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ["@vue/app"]
+  presets: [["@vue/app",{useBuiltIns: false}]],
```

Comment:
by default, `useBuiltIns` is `false`, what does this change help in general? 
https://babeljs.io/docs/en/babel-preset-env
Don't we need to add polyfill?

Reply:
The default @vue/app preset we use (that's part of vue-CLI I guess) has this configuration enabled, so what we're doing here is forcing it to be false.

<img width="813" alt="Screen Shot 2020-02-27 at 11 40 54" src="https://user-images.githubusercontent.com/59408474/75431867-2b7ea100-5956-11ea-9a3a-a5d3c011f8bb.png">

Two other choices are to include core-js with our desired version, Or to force a peer-dependency of the specified version, the latter is a no-go if we want to a broad spectrum of builds

Reply:
@mayashavin  - I added core-js@2.6.9 as a dependency and removed the builtins, that should solve our problems while keeping this library self-contained and not dependent on the consuming project